src/librados/librados_c.cc

```diff
@@ -2597,6 +2600,7 @@ static void rados_aio_getxattrs_complete(rados_completion_t c, void *arg) {
     cdata->it = 0;
     cdata->user_completion.finish(0);
   }
+  cdata->operational_completion->put();
```

Comment:
@aclamk Adam, i think it's not an intuitive fix. probably an alternative is to set the refcount of `AioCompletionImpl` to 0 when it is created. as we always increase its refcount right after capturing its strong reference. in this case, `C_AioCompleteAndSafe` calls `c->get()` when assigning `cc` to `this->c`.

Reply:
@tchaikov Thanks! I did not understand that context passed to completion is the one used in invocation.